server/util.go

```diff
@@ -271,12 +272,13 @@ func AddCommands(rootCmd *cobra.Command, defaultNodeHome string, appCreator type
 		ShowAddressCmd(),
 		VersionCmd(),
 	)
+
 	startCmd := StartCmd(appCreator, defaultNodeHome)
 	addStartFlags(startCmd)
 
 	rootCmd.AddCommand(
 		startCmd,
-		UnsafeResetAllCmd(),
+		tmcmd.ResetAllCmd,
```

Comment:
Should we add the reset state command as well? (The new command that TM introduices)

Reply:
Yup, I'll add that.